volumina/croppingMarkers.py

```diff
@@ -47,12 +65,13 @@ def set_roi_3d(self, roi):
         # Set the extents as a roi
         self.set_crop_extents( zip( *roi ) )
 
-    def set_volume_shape_3d(self, shape3d):
-        # Since the volume size changed, 
+    def set_volume_shape_3d(self, starts, stops):
+        # Since the volume size changed,
```

Comment:
Since you changed this function signature to accept a roi instead of a shape, I think you should also change the function name.